src/v3/components/reminders/reminders.types.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+export type ReminderItemProps = {
+  title: string;
+  description: string;
+  link: string;
+};
```

Comment:
Seems like this type is duplicated. Please refactor it so we will have one ReminderItemProps type. Additionally, since Reminders and ReminderItem are connected, what do you think about moving them to one 'reminders' folder? (just a suggestion, you can ignore it)